TVCROLLDAP.s

```diff
@@ -150,7 +150,7 @@
 #	SECOND INTERVALS) WILL BENEFIT FROM THE CONVERGENT NATURE OF THE
 #	APPROXIMATION.
 #
-# FOR LARGE OGAERROR THE TANGENT INTERSECTS +-MINLIM SWITCH BOUNDRY BEFORE
+# FOR LARGE OGAERROR THE TANGENT INTERSECTS +-MINLIM SWITCH BOUNDARY BEFORE
```

Comment:
`BOUNDRY` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0987.jpg)